generator/Generator/MapGenerator.php

```diff
@@ -106,7 +107,15 @@ public function generate(): void
     private static function scanExistingModelEndpointClasses(string $directory): array
     {
         $map = [];
-        $recursiveIteratorIterator = new RecursiveIteratorIterator(new RecursiveDirectoryIterator($directory));
+
+        try {
+            $directoryIterator = new RecursiveDirectoryIterator($directory);
+        } catch (\UnexpectedValueException) {
+            mkdir($directory, 0755, true);
```

Comment:
Added when new APIs are added, like `Origin Errors API`, that do not have existing directories yet.